### ISSUE TITLE
Use query endpoint

### ISSUE THREAD

**Issue by yaacov** [MohawkTSDB/mohawk-webui-graph#3]
Currently we use the `raw` endpoint that is limited to one metric `id` [1]
Using the `query` endpoint enables more advanced searches (e.g. multiple ids and search by tags)

examples:
```
# get mem and cpu by id
curl http://127.0.0.1:8080/hawkular/metrics/gauges/raw/query -d "{\"ids\":[\"free_memory\", \"cpu_usage\"],\"start\":\"-2h\"}"

# search metrics by label regexp, and aggregate results for each 15 minutes
curl http://127.0.0.1:8080/hawkular/metrics/gauges/raw/query -d "{\"tags\":\"name:.*cpu.*|.*memory.*\",\"start\":\"-15mn\",\"bucketDuration\":\"15mn\"}"
```

[1] https://github.com/MohawkTSDB/mohawk-webui-graph/blob/master/src/App.js#L35

**Comment by nimrodshn** [comment]
nice! I didn't know we had that!
I'll add that ASAP 👍

**Comment by nimrodshn** [comment]
resolved in #14
